packages/app/cypress/e2e/specs_list_e2e.cy.ts

```diff
@@ -0,0 +1,88 @@
+describe('App: Spec List (E2E)', () => {
+  beforeEach(() => {
+    cy.scaffoldProject('cypress-in-cypress')
+    cy.openProject('cypress-in-cypress')
+    cy.startAppServer('e2e')
+    cy.visitApp()
+  })
+
+  it('shows the "Specs" navigation as highlighted in the lefthand nav bar', () => {
+    cy.findByLabelText('Sidebar').within(() => {
```

Comment:
<img width="296" alt="Screenshot 2022-02-07 at 12 16 47 PM" src="https://user-images.githubusercontent.com/27690333/152838309-ffd08006-281d-438b-b66d-2353487bc793.png">

I was checking the `router-link-active` class since it will only be attached to the `<a>` that is currently selected. Do you think it would be worthwhile to switch to using a data attribute instead?

Reply:
I think switching to a data attribute will tie us less to class specific implementations. That'd be my preference.